generic/secrets/security/detected-azure-storage-key.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
+- id: detected-azure-storage-key
+  pattern-regex: |-
+    [A-Za-z0-9/+=]{88}
```

Comment:
Similar to `detected-azure-service-bus-key`, this looks like it may detect a lot of things. Can we get better accuracy to make sure the finding is an Azure Storage Key?